portfolio/src/main/webapp/index.html

```diff
@@ -76,66 +85,62 @@ <h5 class="card-title"> What's The Dish? </h5>
+                                <p class="card-text"><small class="text-muted"> Intel Cooporation </small></p>
+                                <p class="card-text"> Designed and implemented two AI machine learning models (SVM and logistics regression) to answer the question "Can we predict, using Artificial Intelligence algorithms, if an individual will recall their dream?" </p>
+                            </div>
+                            </div>
+                        </div>
+                    </div>                
                 </div>
             </div>
-            </div>
-            </br>            
+            </br>
 
-            <div class="card">
-                <div class="card-header"> Updates </div>
+            <div class="card" id="activities">
+                <div class="card-header"> Memberships and Outreach </div>
                 <div class="card-body">
-                    <div class="card">
-                        <div class="card-body">
-                            <h5 class="card-title"> S.T.E.P. Internship Week 2 </h5>
-                            <p class="card-text"><small class="text-muted">25 May 2020</small></p>
-                            <p class="card-text">This week I started with the individual projects set up for S.T.E.P Interns. I was also able to attend some official Google team meetings and some 1 on 1 meetings with my hosts. </p>
+                    <div class="row">
+                        <div class="col-sm-6">
+                            <div class="card">
+                            <div class="card-body">
+                                <h5 class="card-title">Memberships</h5>
+                                <p class="card-text">
+                                    <ul> 
+                                        <li> Phi Sigma Rho (Engineering Sorority) </li>
+                                        <li> Medallion Leadership and Service Program </li>
+                                    </ul>
+                                </p>
+                            </div>
+                            </div>
                         </div>
-                        </br>
-                    </div>                   
+                        <div class="col-sm-6">
+                            <div class="card">
+                            <div class="card-body">
+                                <h5 class="card-title"> Community Outreach </h5>
+                                <p class="card-text">
+                                    <ul>
+                                        <li> Connect2STEM Outreach Event </li>
+                                        <li> Produce Rescue with Borderlines </li>
+                                    </ul>
+                                </p>
+                            </div>
+                            </div>
+                        </div>
+                    </div>
                 </div>
-            </div>
+            </div>          
```

Comment:
I think there  is a bunch of white space at the end of this line here that can be removed.